### DISCUSSION TITLE
No connection when I stop Openvpn from ui

### DISCUSSION THREAD

**Discussion by malex2005** [RaspAP/raspap-webgui#924] [Q&A] [answered]
Hello,

My RaspAp is going strong. Specially with Openvpn, it is great. Thank you very much :)

My little issue is when I stop openvpn from the web interface, the internet connexion is lost. Wifi is still up.
The tun0 is removed so as the network routes so it seems normal there. A reboot of the pi with openvpn stopped does not solve the problem
When I restart openvpn all is back to normal.

Anything I am missing out?

Al

**Comment by billz** [comment, marked as answer]
That's an odd one. It's possible that the route created for the `tun0` interface isn't being properly removed. To check for this, drop to the shell and execute `journalctl | grep openvpn`. For example, this sequence is typical when OpenVPN is stopped:

```
raspberrypi sudo[22418]: www-data : TTY=unknown ; PWD=/var/www/html ; USER=root ; COMMAND=/bin/systemctl stop openvpn-client@client
raspberrypi openvpn[22322]: event_wait : Interrupted system call (code=4)
raspberrypi openvpn[22322]: /sbin/ip route del 192.145.80.123/32
raspberrypi openvpn[22322]: /sbin/ip route del 0.0.0.0/1
raspberrypi openvpn[22322]: /sbin/ip route del 128.0.0.0/1
raspberrypi openvpn[22322]: Closing TUN/TAP interface
raspberrypi openvpn[22322]: /sbin/ip addr del dev tun0 10.7.2.3/24
raspberrypi openvpn[22322]: SIGTERM[hard,] received, process exiting
raspberrypi systemd[1]: openvpn-client@client.service: Succeeded.
```

This restores the default routing and normal internet access via the AP. You can also try executing ` ip r` before and after starting OpenVPN and compare the results.

**Reply by malex2005** [reply]
Hi

Thanks for your reply! I ran the journalctl command and all looked fine. It turned out my dhcp configuration was at fault as I setup a DNS reachable only with the vpn (pihole on a cloud). I figured this out as a ping to the internet was ok.

Al
